mllib/src/main/scala/org/apache/spark/mllib/tree/configuration/Strategy.scala

```diff
@@ -184,6 +201,7 @@ object Strategy {
   /**
    * Construct a default set of parameters for [[org.apache.spark.mllib.tree.DecisionTree]]
    * @param algo Algo.Classification or Algo.Regression
+   * @since 1.3.0
    */
   def defaultStategy(algo: Algo): Strategy = algo match {
```

Comment:
This has been renamed to `defaultStrategy` (and rightly so). Please rebase with master.